src/network/netdev/netdev-gperf.gperf

```diff
@@ -162,6 +162,7 @@ MACsecReceiveAssociation.Activate,        config_parse_macsec_sa_activate,
 Tun.OneQueue,                             config_parse_bool,                         0,                             offsetof(TunTap, one_queue)
 Tun.MultiQueue,                           config_parse_bool,                         0,                             offsetof(TunTap, multi_queue)
 Tun.PacketInfo,                           config_parse_bool,                         0,                             offsetof(TunTap, packet_info)
+Tun.VNetHeader,                           config_parse_bool,                         0,                             offsetof(TunTap, vnet_hdr)
```

Comment:
The commit title implies that this was just omitted by mistake. But afaics, support was added just for tap in back in f5f07dbf06f6df9d12e092c05ad8cfbe244d203f on purpose. So shouldn't the commit just say "Allow IFF_VNET_HDR to also be set for tun devices"? What is the enabling this?